splunk_base/README.md

```diff
@@ -0,0 +1,3 @@
+# StackState Splunk client
```

Comment:
not a client anymore but the base to provide Splunk client and helper/utils with config provider kind of.